File: app/src/main/java/com/wealoha/social/beans/region/RegionResult.java
package com.wealoha.social.beans.region;

import java.io.Serializable;
import java.util.ArrayList;

import com.wealoha.social.beans.ResultData;

/**
 * @author:sunkist
 * @see:
 * @since:
 * @description
 * @copyright wealoha.com
 * @Date:2015年7月9日
 */
public class RegionResult extends ResultData implements Serializable {

	/**
	 * 
	 */
	private static final long serialVersionUID = 1853858837485072423L;
	public boolean filterEnable;
	public int filterAgeRangeStart;
	public int filterAgeRangeEnd;
	public String filterRegion;
	public ArrayList<Regions> regions;
	public ArrayList<String> selectedRegion;
}

File: app/src/main/java/com/wealoha/social/beans/RegionResult.kt
package com.wealoha.social.beans

import java.io.Serializable
import java.util.*

/**
 * @author:sunkist
 * @see:
 * @since:
 * @description
 * @copyright wealoha.com
 * @Date:2015年7月9日
 */
class RegionResult : ResultData(), Serializable {
    var filterEnable = false
    var filterAgeRangeStart = 0
    var filterAgeRangeEnd = 0
    var filterRegion: String? = null
    var regions: ArrayList<Regions>? = null
    var selectedRegion: ArrayList<String>? = null

    companion object {
        /**
         *
         */
        private const val serialVersionUID = 1853858837485072423L
    }
}